core-it-suite/src/test/java/org/apache/maven/it/MavenITmng6090CIFriendlyTest.java

```diff
@@ -19,7 +19,6 @@
  * under the License.
  */
 
-import org.apache.maven.it.Verifier;
```

Comment:
unrelated change

Reply:
True. We were investigating that test case and probably the IDE thought it'd be good to cleanup unused imports. Prefer to have it reverted? I think it won't hurt either....

Reply:
It will look very weird if we go through the commit logs and see the list of files touched by this commit. It scares me if I see MNG6090 between MNG4660

Reply:
Don't be scared, it's been reverted already.